tools/rpkg/src/relational.cpp

```diff
@@ -184,7 +185,16 @@ external_pointer<T> make_external(const string &rclass, Args &&...args) {
 		cond = make_unique<ConjunctionExpression>(ExpressionType::CONJUNCTION_AND, move(cond_args));
 	}
 
-	auto res = std::make_shared<JoinRelation>(left->rel, right->rel, move(cond), JoinType::INNER);
+	auto join_type = JoinType::INNER;
+	std::transform(join.begin(), join.end(), join.begin(), [](unsigned char c) { return std::tolower(c); });
```

Comment:
I don't think we need that, enum arguments as characters are typically case-sensitive.

```suggestion
```